modules/simulation/JsYetiSimulator/index.js

```diff
@@ -268,330 +268,528 @@ function dutyCycleToAmps(dc) {
 
 function error_BrownOut(mod, raise) {
   if (!active_errors.BrownOut) {
-    mod.provides.board_support.raise.evse_board_support_BrownOut('Simulated fault event', 'High');
+    let error = mod.provides.board_support.error_factory.create_error(
+      'evse_board_support/BrownOut',
+      '',
+      'Simulated fault event',
+      'High'
+    );
+    mod.provides.board_support.raise_error(error);
     active_errors.BrownOut = true;
   } else if (!raise && active_errors.BrownOut) {
-    mod.provides.board_support.request_clear_all_of_type.evse_board_support_BrownOut();
+    mod.provides.board_support.clear_error('evse_board_support/BrownOut', true);
```

Comment:
Unfortunately there is no create_error function without sub_type parameter, but to clear an error it would also possible to do so:
```suggestion
    mod.provides.board_support.clear_error('evse_board_support/BrownOut');
```
This is same as
```
mod.provides.board_support.clear_error('evse_board_support/BrownOut', false);
````
and will clear at maximum one error.

See also https://everest.github.io/nightly/general/01_framework/error_framework.html#id13

It is your decision what to use here, but I would recommend to use my suggestion in case of not using sub_types here


... same for the rest

Reply:
Changed this